docs/user-guide/quickstart.md

```diff
@@ -83,7 +85,23 @@ Log in to the Apple Developer Portal (https://developer.apple.com/account), and
 
 You now have the vendor side of the certificate flow complete, and you need to complete the customer side of this flow, with the help of the vendor cert. 
 
-Sign a push certificate request with the vendor certificate. This step uses the private key you created above, so specify the same password to be able to decrypt it. 
+## Customer Process
+
+### MicroMDM Push Certificate Request[¶](http://localhost:1313/micromdm/create-dep-certificates/#micromdm-push-certificate-request)
```

Comment:
It's weird that this is a localhost link. Was this meant to link somewhere else?